.github/workflows/c-cpp.yml

```
@@ -6,8 +6,22 @@
+  __bin: dbs26
+  __builddir: ${{ github.workspace }}/build
+  __out: ${{ github.env.__builddir }}/${{ github.env.__bin }}
+  __destdir: ${{ github.env.__builddir }}/out
+  __srcdir: ${{ github.workspace }}/src
+  __exe: ${{ github.env.__out }}
+
 jobs:
   build-ubuntu-24_04:
+    env:
+      __msvc_git: ${{ github.workspace }}/msvc
+      __msvc_sdk: ${{ github.env.__msvc_git }}/sdk
```

actionlint (1.7.4)

receiver of object dereference "__msvc_git" must be type of object but got "string"

(expression)